pc/http/http_conformance.py

```
@@ -0,0 +1,334 @@
+
+    # 3. Total size, from a one-byte range -- the client has no HEAD and must not guess.
+    status, hdrs, body = get(conn, iso_path, (0, 0))
+    if status != 206:
+        res.bad("size probe", "expected 206 for bytes=0-0, got {}".format(status))
+        print("\n{} passed, {} failed, {} skipped".format(res.passed, res.failed, res.skipped))
+        return 1
+    cr = parse_content_range(hdrs.get("content-range"))
+    if cr is None or cr[2] is None:
+        res.bad("size probe", "no usable total in Content-Range: {!r}".format(
+            hdrs.get("content-range")))
+        print("\n{} passed, {} failed, {} skipped".format(res.passed, res.failed, res.skipped))
```

Ruff (0.16.3)

[warning] Use f-string instead of `format` call

Convert to f-string

(UP032)